test/api/bundle.test.ts

```diff
@@ -43,44 +45,33 @@ test('write', () => {
     packageDir: pkg.dir,
     entryPoints: [pkg.entrypoint],
     allowedLicenses: ['Apache-2.0', 'MIT'],
+    // makes the bundle snapshot determenistic
+    sourcemap: false,
   });
 
   const bundleDir = bundle.write();
 
   expect(fs.existsSync(path.join(bundleDir, pkg.entrypoint))).toBeTruthy();
   expect(fs.existsSync(path.join(bundleDir, 'package.json'))).toBeTruthy();
-  expect(fs.existsSync(path.join(bundleDir, 'THIRD_PARTY_LICENSES'))).toBeTruthy();
```

Comment:
This should never have been here, the tests used to mistakenly create empty attribution files, and this test specifically doesn't do validation.